pyproject.toml

```diff
@@ -84,6 +84,7 @@ testpaths = [
 ]
 filterwarnings = [
     "error",
+    "ignore::DeprecationWarning",
```

Comment:
In the PR description you write 
> update code to not raise deprecation warnings

since the code was updated to not use the "older typing syntax", can we please remove this suppression/ignore rule? I ran pytest locally and get no warnings with this line removed.

Reply:
this is pydantic 1.x.x causing this .... see https://github.com/BeanieODM/beanie/actions/runs/11526533446/job/32090846596?pr=1062

and if we remove it then we can remove it

Reply:
Okay @07pepa, my mistake then. Good point.